### DISCUSSION TITLE
Incorrect calculation of the percentage of translation into other languages.

### DISCUSSION THREAD

**Discussion by DxUp** [directus/directus#15378] [Archive]
Hi,

Please help with this:

If not all fields in the content are translatable, this error occurs. In the general list of content and in the content itself, the percentage of translations into other languages is not the same.

![Screen-001-(03-09-2022_08)](https://user-images.githubusercontent.com/109763298/188257415-b2515585-c853-4f6f-9fb2-3b64879cd235.jpg)
![Screen-002-(03-09-2022_08)](https://user-images.githubusercontent.com/109763298/188257416-22246acc-d94b-4d67-9b4e-87cff67c815a.jpg)

**Comment by rijkvanzanten** [comment]
@Nitwel thoughts?
